main.js

```diff
@@ -169,3 +171,7 @@ function buttonReset(event) {
     }
   }
 }
+
+function resetDataModel() {
+  newOutfit = new Outfit(Date.now());
+}
```

Comment:
This creates a new object with a new date that gets fired after we save the current outfit to local storage